autopilot/interface.go

```diff
@@ -39,7 +39,7 @@ type Node interface {
 // Channel is a simple struct which contains relevant details of a particular
 // channel within the channel graph. The fields in this struct may be used a
 // signals for various AttachmentHeuristic implementations.
-type Channel struct {
+type LocalChannel struct {
```

Comment:
nit: godoc needs an update too.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/4666/commits/d3ccbd174ac53ca670c84c8c8a9298c8d5457b83

Reply:
nit: s/may be used a/may be used as